Source/WebKit/PlatformGTK.cmake

```diff
@@ -694,62 +730,57 @@ add_custom_target(WebKit-forwarding-headers
     COMMAND ${PERL_EXECUTABLE} ${WEBKIT_DIR}/Scripts/generate-forwarding-headers.pl --include-path ${WEBKIT_DIR} --output ${FORWARDING_HEADERS_DIR} --platform gtk --platform soup
 )
 
-# These symbolic link allows includes like #include <webkit2/WebkitWebView.h> which simulates installed headers.
+# These symbolic link allows includes like #include <webkit/WebkitWebView.h>, which simulates installed headers.
 add_custom_command(
-    OUTPUT ${WebKit2Gtk_FRAMEWORK_HEADERS_DIR}/webkit2
+    OUTPUT ${WebKitGTK_FRAMEWORK_HEADERS_DIR}/webkit
     DEPENDS ${WEBKIT_DIR}/UIProcess/API/gtk
-    COMMAND ln -n -s -f ${WEBKIT_DIR}/UIProcess/API/gtk ${WebKit2Gtk_FRAMEWORK_HEADERS_DIR}/webkit2
-)
-add_custom_command(
-    OUTPUT ${WebKit2Gtk_DERIVED_SOURCES_DIR}/webkit2
-    DEPENDS ${WebKit2Gtk_DERIVED_SOURCES_DIR}/webkit
-    COMMAND ln -n -s -f ${WebKit2Gtk_DERIVED_SOURCES_DIR}/webkit ${WebKit2Gtk_DERIVED_SOURCES_DIR}/webkit2
```

Comment:
I'm just now realizing that removing this symlink may have made things more difficult than necessary by forcing the need for extra #ifdefs in the .cpp files. That's probably OK though.

Reply:
This doesn't actually matter. It only affects including webkit2/webkit2.h vs. webkit/webkit.h, and a symlink is not enough there because the actual name of the header is different.